### PR TITLE
Improved directed scans

### PR DESCRIPTION
**Pull request by alongd**
The feature of running scans directed by ARC (as opposed to scans managed bv the ESS) is improved (it suffered some functionality loss by the job PR #504)

### CODE REVIEW

File: arc/common.py [hunk 1]
```diff
@@ -482,6 +482,25 @@ def globalize_path(string: str,
     return string
 
 
+def delete_check_files(project_directory: str):
+    """
+    Delete ESS checkfiles. They usually take up lots of space and are not needed after ARC terminates.
+    Pass ``True`` to the ``keep_checks`` flag in ARC to avoid deleting check files.
```
Comment (kfir4444) [inline arc/common.py:488]: Does this required here? I do not think users commonly check `common.py`

File: arc/common.py [hunk 2]
```diff
@@ -482,6 +482,25 @@ def globalize_path(string: str,
     return string
 
 
+def delete_check_files(project_directory: str):
```
Comment (kfir4444) [inline arc/common.py:485]: Untested function

File: arc/job/adapters/ts/gcn_ts.py [hunk 3]
```diff
@@ -37,6 +37,8 @@
 
 TS_GCN_PYTHON = settings['TS_GCN_PYTHON']
 
+DIHEDRAL_INCREMENT = 10
```
Comment (kfir4444) [inline arc/job/adapters/ts/gcn_ts.py:40]: Why is this only here and not in `autotst_ts.py` and the others?

File: arc/job/adapters/ts/heuristics.py [hunk 4]
```diff
@@ -155,7 +157,9 @@ def __init__(self,
         self.conformer = conformer
         self.constraints = constraints or list()
         self.cpu_cores = cpu_cores
+        self.dihedral_increment = dihedral_increment or DIHEDRAL_INCREMENT
```
Comment (kfir4444) [inline arc/job/adapters/ts/heuristics.py:160]: Here the `DIHEDRAL_INCREMENT = 30`

**Review by kfir4444 — COMMENTED** [review]
Thanks @alongd for this PR!
please see some comments.

File: arc/job/adapters/ts/heuristics.py (hunk 4 above)
Comment (alongd) [reply arc/job/adapters/ts/heuristics.py:160]: Yes, in heuristics it is used differently than in GCN, and usually this default value is being used (i.e., ARC doesn't alter it)

File: arc/common.py (hunk 1 above)
Comment (alongd) [reply arc/common.py:488]: This function was copied as is from main to common.
I think that advanced users could look at it before running something so destructive.

**Pull request merged**